.github/workflows/macos.yml

```diff
@@ -71,9 +84,36 @@ jobs:
 
       - run: make
 
+      # Launchable requires Python and Java
+      # https://www.launchableinc.com/docs/resources/cli-reference/
+      - name: Set up Python
+        uses: actions/setup-python@871daa956ca9ea99f3c3e30acb424b7960676734 # v5.0.0
+        with:
+          python-version: "3.10"
+      - name: Set up JDK 17
+        uses: actions/setup-java@7a445ee88d4e23b52c33fdc7601e40278616c7f8 # v4.0.0
```

Comment:
This part is also the same with the setup-python. The commit hash `7a445ee88d4e23b52c33fdc7601e40278616c7f8` is the current latest main branch's commit. Not the same with the `v4.0.0` or `v4` tag. why we don't use the `actions/setup-java@v4.0.0` or `actions/setup-java@v4`?

Reply:
Answered in https://github.com/ruby/ruby/pull/9777/files#r1490623705.

Reply:
Okay. I agreed with following your practice.